tests/check_policy.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# Verify that the policy is correct. If you want to avoid commiting invalid
+# changes, just do:
+#   cp -a tests/check_policy.sh .git/hooks/pre-commit
+
+cf_promises=""
+if type cf-promises >/dev/null 2>&1; then
+  cf_promises="cf-promises";
+elif [ -x "../core/cf-promises/cf-promises" ]; then
+  cf_promises="../core/cf-promises/cf-promises"
```

Comment:
adding --full-check and --eval-functions gives a deeper check.

Reply:
Nice! Added.